services/server/server.go

```diff
@@ -50,6 +50,8 @@ import (
 	"github.com/percona/pmm-managed/utils/envvars"
 )
 
+const timeout = 10 * time.Second
```

Comment:
Let's use a better name. `platformAPITimeout`?

Reply:
Seems that `platformAPITimeout` is good enough. Renamed.